### PR TITLE
Add explicit lifecycle to labs

### PR DESCRIPTION
**Pull request by ryjones**
This makes explicit the steps to follow after a lab is dormant.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,13 @@ not the case, you will need to bring your code by squashing all of
 your commits into a single first commit made against your new lab
 repo with your sign-off.
 
+## Lifecycle
+
+Archiving
```
Comment (tkuhrt) [inline README.md:50]:
```suggestion
## Archiving

```

File: README.md [hunk 2]
```diff
@@ -45,6 +45,13 @@ not the case, you will need to bring your code by squashing all of
 your commits into a single first commit made against your new lab
 repo with your sign-off.
 
+## Lifecycle
```
Comment (tkuhrt) [inline README.md:48]:
```suggestion
```

**Review by tkuhrt — APPROVED** [review]
This looks okay to me. I made a few suggested edits. Wondering if we can create a action to run automatically on all labs that would do the check and create a PR automatically.

**Review by lehors — APPROVED** [review]
LGTM pending the fix Tracy suggested.

**Pull request merged**